Makefile

```diff
@@ -40,16 +40,20 @@ lint: ## Run golangci-lint against code.
 ##@ Build
 
 .PHONY: build
-build: build-darwin build-linux build-windows ## Build gardenctl binary for darwin, linux and windows.
+build: build-darwin-amd64 build-darwin-arm64 build-linux build-windows ## Build gardenctl binary for darwin, linux and windows.
```

Comment:
let's also adapt the other make targets and add the arch there

Reply:
I forgot those